include/envoy/server/configuration.h

```diff
@@ -73,7 +73,7 @@ class Main {
    * @return std::chrono::milliseconds the time interval between flushing to configured stat sinks.
    *         The server latches counters.
    */
-  virtual std::chrono::milliseconds statsFlushInterval() PURE;
+  virtual const std::chrono::milliseconds& statsFlushInterval() PURE;
```

Comment:
why return a const ref here?  Most of the other time-returning things I see in Envoy are returning by value.